packages/components/organisms/f-form/src/components/Form.vue

```diff
@@ -1,46 +1,104 @@
 <template>
-    <div
+    <form
+        method="post"
         :class="$style['c-form']"
-        data-test-id="form-component">
-        {{ copy.text }}
-    </div>
+        data-test-id="form-component"
+        @submit.prevent="onFormSubmit">
+        <form-field
+            v-for="field in formData.formFields"
+            :key="`${field.name}-field`"
+            v-bind="fieldProps(field)"
+            @input="updateField({ fieldName: field.name, value: $event })" />
+
+        <f-button
+            button-type="primary"
+            button-size="large"
+            :class="$style['c-form-submitButton']"
+            is-full-width
+            action-type="submit"
+            data-test-id="submit-button"
+            :is-loading="isFormSubmitting">
+            {{ formData.buttonText }}
+        </f-button>
+    </form>
 </template>
 
+
 <script>
+import FButton from '@justeat/f-button';
+import '@justeat/f-button/dist/f-button.css';
```

Comment:
Just a note that when you move these to be `peerDependencies`, you won't need to include the CSS here, as it'll get pulled in as part of the page level component (i.e. Checkout) instead. You'll need the styles included in any storybook files you create for testing purposes though.

Reply:
Awesome, Thanks